src/tokenizer.rs

```diff
@@ -168,7 +168,6 @@ pub enum Comment<'a> {
     },
     MultiLine {
         content: Cow<'a, str>,
-        depth: u32,
```

Comment:
Ah, multi-line comments and strings have a tricky feature represented here where you can have any number of equals signs between your brackets to create more powerful comments and strings!

```lua
--[===[
this comment is [[so]] powerful
]===]
```

It's usually used to embed Lua source inside Lua source. This value tries to represent how many equals signs are in a given comment. This might complicate using a single regex to parse comments.

Reply:
Got it.